pyexasol/connection.py

```diff
@@ -1206,16 +1206,16 @@ def _get_ws_options(self):
         }
 
         if self.options["encryption"]:
```

Comment:
Move info to a more apparent level (or change implementation)? As in `_get_ws_options` of class, it's not shown in the generated documentation. Could set in `__init__` or more directly expose.